core/platform/datastore/gae_datastore_services.py

```diff
@@ -93,6 +93,13 @@ def put_multi(models):
     Returns:
         list(str). A list with the stored keys.
     """
+    for model in models:
+        # We can remove this once all the models have inherited from
+        # BaseHumanMaintainedModel.
+        if (
+                'BaseHumanMaintainedModel' in
+                [x.__name__ for x in inspect.getmro(type(model))]):
+            model.put_async = model.put_async_for_bot
```

Comment:
I don't think that this is the correct solution, it hides a lof of stuff and also modifying class/models dynamically can be quite dangerous. Also regarding the comment, we do not expect to use `BaseHumanMaintainedModel` for all models even after we are done with this whole issue.

Reply:
I have added a lint check instead of this solution. PTAL.